### DISCUSSION TITLE
Latest update breaks code completion in docker container?

### DISCUSSION THREAD

**Discussion by dspyz-matician** [community/community#123307] [Copilot Conversations]
### Select Topic Area

Bug

### Body

The latest copilot update (1.89 as of two hours ago) seems to cause completions to not work using the Remote Development extension for working inside a docker container (using the Docker extension).
When I ask for completions, I get "command 'github.copilot.generate' not found"
If I roll back to 1.88 or work outside the docker container I get completions back

**Comment by raideer** [comment]
Same issue when using Copilot in WSL

**Comment by MustaphaDG** [comment]
I'm facing the identical issue.

**Comment by ZuhairS** [comment]
I have the same issue when using the Remote Development extension to connect via tunnel. Copilot icon is missing. Works fine in local and remote machines but not through the tunnel.

**Comment by justindoran** [comment]
Same. This happens in all Codespaces. Switched back to version v 1.188.0 and it immediately worked again on reload. v 1.189.0 does work on non-dockerized environments. Reinitiating the auth-flow and reinstalling the extension does not fix 1.189's problems.

**Comment by dspyz-matician** [comment]
Seems fixed in 1.190
Thanks!
